docs/getting_started/why_use_GTFS.md

```diff
@@ -1,42 +1,51 @@
-- [Flexible services features](/getting_started/features/flexible_services), offering demand-responsive transit options, like dial-a-ride and paratransit services; 
-- [Pathways features](/getting_started/features/pathways), displaying vital accessibility information for rider navigation in stations. 
-- And [many more](/getting_started/features/overview).
+GTFS fosters collaboration between transit agencies by enabling easy data exchange, promoting smoother service integration, and information sharing. Here's how GTFS simplifies data sharing:
+
+- **Ease of Integration:** GTFS makes it easy for agencies to get started with a simple data structure using common file formats like .txt and GeoJSON, fostering collaboration and interoperability.
+
+## GTFS can probably do more than you think
 
-Additionally, GTFS data is now being used by a variety of software applications including data visualization and analysis tools for planning and research. Having up-to-date and high quality data provides accurate transit information not just to riders, but also to planners and policymakers who are able to better understand how transit is being used in their communities. 
+While trip planning is a core function, GTFS offers additional features that enhance the transit experience:
 
-## Open Data means more opportunity and choices
+- **Fares:** GTFS can include fare information for accurate cost breakdowns across routes and journey options.
+- **Flexible services:** GTFS can describe demand-responsive options like dial-a-ride, route deviations, and other services that do not follow the common behavior of scheduled and/or fixed service.
+- **Pathways:** GTFS can model large transit stations, helping riders navigate from station entrances and exits to the location where they board or disembark from a transit vehicle.
 
-GTFS is an Open Standard. This means that agencies can make information available using any of the tools which already support GTFS (including simple text editing or a spreadsheet). Open standards lead to the creation of data that can be easily shared. The feeds can be used by trip planners such as Google, Apple, Transit App, Open Trip Planner, and even apps created by riders. Anyone who wants to provide accurate and up-to-date transit information can use GTFS feed to do so.
+[:octicons-search-16: Learn more about GTFS Features](../features/overview){ .md-button }
 
-Some riders like to use different apps depending on their needs—having GTFS lets riders choose what trip planning app suits them best. Some apps may be more accessible or better at providing information for riders with disabilities, some may be simpler and easier to use, and sometimes riders just want the newest app.
+## Improved Rider Experience
 
-## Maintained by the community
+GTFS empowers riders with accurate schedules, real-time updates, and a wider choice of apps, leading to reduced wait times and informed travel decisions. Here's how GTFS enhances the rider experience:
```

Comment:
I'm not too sure about this, but I think that maybe we can remove the segment that aludes to a wider choice of apps, since it's being covered in the next section.